src/tests/test_process.py

```diff
@@ -56,6 +92,17 @@ def _stop_process(cls, timeout=5):
             return False
         return True
 
+    @classmethod
+    def _kill_process(cls, timeout=5):
+        try:
+            cls.process.send_signal(signal.SIGKILL)
+            cls.process.wait(timeout)
+        except psutil.NoSuchProcess:
+            return True
+        except psutil.TimeoutExpired:
+            raise OSError("Process is stuck")
```

Comment:
Maybe `self.fail()` here?

Reply:
ok

Reply:
`_kill_process` is a class method so you'd have to use `cls.fail()` and I don't think that would work correctly. The documentation doesn't say what happens if there are exceptions in `tearDownClass`, only in `tearDown`. Looking at [unittest source](https://github.com/python/cpython/blob/0f221d09cad46bee38d1b7a7822772df66c53028/Lib/unittest/suite.py), it looks like an exception in `tearDownClass` also adds a new error entry in the totals, just like `tearDown` is supposed to. So I think a more traditional exception like `OSError` is a better choice here. Perhaps an even better choise is to simply delete this `except` and let the `psutil.TimeoutExpired` propagate up.